deployment/paiLibrary/clusterObjectModel/paiObjectModel.py

```diff
@@ -194,7 +194,10 @@ def serviceParse(self):
             serviceDict["clusterinfo"]["docker-registry-info"]["docker-tag"]
         serviceDict["clusterinfo"]["dockerregistryinfo"]["secretname"] = \
             serviceDict["clusterinfo"]["docker-registry-info"]["secret-name"]
-
+        serviceDict["clusterinfo"]["api-server"]["tls_access"] = False
+        if ("api-server" in serviceDict["clusterinfo"]) and ("tls_access" in serviceDic["clusterinfo"]["api-server"]):
+            serviceDict["clusterinfo"]["apiserver"] = serviceDic["clusterinfo"]["api-server"]
```

Comment:
what this means? change the name? I see template uses key of `api-server` instead of `apiserver`